<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="first_name_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="First_name_input">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">z-index: 3;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Last_name_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Last name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_name_input">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> setZValue:1;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="policeman">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>351</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logos/policeman.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>policeman</zorder>
   <zorder>first_name_label</zorder>
   <zorder>First_name_input</zorder>
   <zorder>Last_name_label</zorder>
   <zorder>last_name_input</zorder>
   <zorder>login_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLog_in">
    <property name="title">
     <string>Log in</string>
    </property>
    <addaction name="separator"/>
    <addaction name="log_as_admin"/>
    <addaction name="log_as_ticketsman"/>
    <addaction name="log_as_driver"/>
   </widget>
   <addaction name="menuLog_in"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="log_as_admin">
   <property name="text">
    <string>Log as Admin</string>
   </property>
  </action>
  <action name="log_as_ticketsman">
   <property name="text">
    <string>Log as Ticketsman</string>
   </property>
  </action>
  <action name="log_as_driver">
   <property name="text">
    <string>Log as Driver</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
